examples/testing/ping_pong.cpp

```diff
@@ -1,13 +1,17 @@
-#define CAF_SUITE ping_pong
 
-#include "caf/test/dsl.hpp"
-#include "caf/test/unit_test_impl.hpp"
+// This file is part of CAF, the C++ Actor Framework. See the file LICENSE in
+// the main distribution directory for license terms and copyright or visit
+// https://github.com/actor-framework/actor-framework/blob/master/LICENSE.
 
-#include "caf/all.hpp"
+#include "caf/test/caf_test_main.hpp"
+#include "caf/test/fixture/deterministic.hpp"
+#include "caf/test/test.hpp"
+
+#include "caf/event_based_actor.hpp"
 
 using namespace caf;
+using namespace caf::test;
 
-// --(rst-ping-pong-begin)--
```

Comment:
These markers are for including this block in the manual: https://actor-framework.readthedocs.io/en/stable/Testing.html#deterministic-testing.

Please put them back in.